release-notes.md

```diff
@@ -0,0 +1,27 @@
+## Changes by Kind
```

Comment:
are the enforce release notes supposed to be here? :p

Reply:
lol, oh my god, how i added that, it is magic